bin/migrations/force_create_tables.py

```diff
@@ -1,207 +0,0 @@
-import argparse
```

Comment:
If I'm reading the Git history right, `gdc_postgres_admin` is a rewrite of this script, so even apart from the "force" thing, it seems like a good idea to get rid of this. Good catch.